src/js/_store/modules/categories.js

```diff
@@ -27,5 +72,30 @@ export default {
         commit('failRequest', { message: err.message });
       });
     },
+    postCategory({ commit, rootGetters, dispatch }, categoryName) {
+      return new Promise((resolve, reject) => {
+        commit('clearMessage');
+        commit('toggleLoading');
+        const data = new URLSearchParams();
+        data.append('name', categoryName);
+        axios(rootGetters['auth/token'])({
+          method: 'POST',
+          url: '/category',
+          data,
+        }).then(() => {
+          commit('toggleLoading');
+          commit('displayDoneMessage', { message: 'カテゴリーを作成したやで' });
```

Comment:
[imo]

もう少し適切な文言でメッセージは表示するようにしましょう

```suggestion
          commit('displayDoneMessage', { message: 'カテゴリーを作成しました' });
```